shopinvader_address/readme/USAGE.rst

```diff
@@ -0,0 +1,11 @@
+[ This file must be present and contains the usage instructions
```

Comment:
It would be nice to explain how addresses works into shopinvader. 1 billing address on the partner itself and 0-n delivery addresses. How and when they can be updated.